env0/data_template.go

```diff
@@ -154,3 +144,32 @@ func dataTemplateRead(ctx context.Context, d *schema.ResourceData, meta interfac
 	//TODO: sshkeys
 	return nil
 }
+
+func getTemplateByName(name interface{}, meta interface{}) (client.Template, diag.Diagnostics) {
+	apiClient := meta.(*client.ApiClient)
+	templates, err := apiClient.Templates()
+	var template client.Template
+
+	if err != nil {
+		return client.Template{}, diag.Errorf("Could not query templates: %v", err)
+	}
+	for _, candidate := range templates {
+		if candidate.Name == name {
+			template = candidate
+		}
```

Comment:
We should error if there's more than one match

Reply:
https://github.com/env0/terraform-provider-env0/pull/41/commits/901aff610f845715919d921520056fcb76d82713